README.md

```diff
@@ -18,13 +21,15 @@ qTox
 * **Mac**: [**download**](#) - Currently unmaintained<br/>
 
 
-<h4>Screenshots</h4>
-Note: The screenshots may not always be up to date, but they should give a good idea of the general look and features
-<img src="http://i.imgur.com/hwGmDeK.png"/>
-<img src="http://i.imgur.com/tmX8z9s.png"/>
+## Screenshots
+**Note:** The screenshots may not always be up to date, but they should give a good idea of the general look and features.
+
+
+![Screenshot 01](https://i.imgur.com/hwGmDeK.png)
+![Screenshot 02](https://i.imgur.com/tmX8z9s.png)
```

Comment:
Again, text size was carefully picked, and this change breaks it.

As for bold ``Note`` - it also doesn't look good.

The only change here that could be applied is markdown-style for screenshots.. although, if screenshots "work" well, then does it really need changing?